parfive/downloader.py

```diff
@@ -364,6 +371,11 @@ def callback(token, future, main_pb):
         """
         timeout = aiohttp.ClientTimeout(**timeouts)
         try:
+            if 'PROXY' in os.environ:
+                kwargs['proxy'] = os.environ['PROXY']
+            if 'PROXY_AUTH' in os.environ:
+                kwargs['proxy_auth'] = os.environ['PROXY_AUTH']
+
             async with session.get(url, timeout=timeout, **kwargs) as resp:
```

Comment:
To test the proxy, we don't actually need to use a proxy. We need to test our library code, not that aiohttp does it's job. What I suggest you do is [mock](https://docs.python.org/3/library/unittest.mock.html) this `session.get` call, and verify that the proxy arguments have been passed to it correctly.